pylint/checkers/strings.py

```diff
@@ -499,8 +497,13 @@ def _check_new_format_specifiers(self, node, fields, named):
                                          node=node)
                         break
 
+                inferred = previous.infer()
+                if inferred is astroid.Uninferable:
```

Comment:
This should be a wrong thing. ``.infer()`` needs to raise or return a generator, never an ``Uninferable`` object.Can you point me to a test that fails with this?